### DISCUSSION TITLE
[Pydantic V2] Can `@field_validator` be used with `@staticmethod`?

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [pydantic/pydantic#6857] [Question]
The following code works with Pydantic v2:
```py
from pydantic import BaseModel, field_validator


class Demo(BaseModel):
    foobar: int

    @field_validator("foobar")
    @staticmethod
    def example_validator(field: int) -> int:
        return field * 2


print(Demo(foobar=42).foobar)
```

Why does documentation [suggest to use `@classmethod`](https://docs.pydantic.dev/latest/usage/validators/#field-validators) instead? What is wrong with `@staticmethod` if I do not access `cls` argument?

**Comment by mariuswallraff** [comment]
I second this question, having to use `@classmethod` and `_ = cls` to appease certain linters (e.g., mypy + vulture) can be fairly annoying
